Applications/ApplicationsLib/ProjectData.h

```diff
@@ -13,7 +13,10 @@
 #ifndef PROJECTDATA_H_
 #define PROJECTDATA_H_
 
+#include <map>
```

Comment:
It's interesting that we didn't have that include before...
Same for string below.